modules/client-configuration/pages/disconnected-setup.adoc

```diff
@@ -1,226 +1,147 @@
-[[client-cfg-reg-with-bootstrap-disconnected]]
-= Disconnected Setup with RMT or SMT (DMZ)
+[[disconnected-setup]]
+= Disconnected Setup
 
-When it is not possible to connect {productname}
-directly or via a proxy to the Internet, a disconnected setup in combination with RMT or SMT is the recommended solution.
+When it is not possible to connect {productname} to the internet, you can use it within a disconnected environment.
 
-In this scenario, RMT or SMT stays in an "`external`" network with a connection to {scc}
- and synchronizes the software channels and repositories on a removable storage medium.
-Then you can separate the storage medium from RMT or SMT, and mount it locally on your {productname}
- server to read the updated data.
+The repository mirroring tool (RMT) is available on {sle}{nbsp}15 and later.
+RMT replaces the subscription management tool (SMT), which can be used on older {sle} installations.
 
+In a disconnected {productname} setup, RMT or SMT uses an external network to connect to {scc}.
+All software channels and repositories are synchronized to a removable storage device.
+The storage device can then be used to update the disconnected {productname} installation.
 
-.Offline Usage Scenario
-[NOTE]
-====
-SMT and RMT are not made for server cascades.
```

Comment:
I can't really work out what this sentence is saying, but after some discussion with a local dev, perhaps it's trying to say that you can't use it if you have a proxy set up? @mcalmer are you able to help me here?